spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -189,7 +189,7 @@ def paginated_collection(*args)
       end
 
       it "should display proper message (including number and not hash)" do
-        expect(pagination.find_by_class('pagination_information').first.content.gsub('&nbsp;',' ')).
+        expect(pagination.find_by_class('pagination-information').first.content.gsub('&nbsp;',' ')).
```

Comment:
Line is too long. [100/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.